odoo/tools/template_inheritance.py

```diff
@@ -138,6 +138,9 @@ def extract(spec):
                         else:
                             comment = content
                     source = copy.deepcopy(spec_content)
+                    for attr in ('t-name', 't-inherit'):
```

Comment:
@aab-odoo @kebeclibre I am definitely willing to remove t-inherit attribute from all templates (maybe when the job is finalized?)
May reasons for that:
- better thing to transport less information
-better thing to transport only pertinent information (avoid useless complexity and reduce the need to know a lot of things at first)
- the web client only need to know the mapping component -> template
- In case of t-inherit-mode="extension", there is a commentary in the template
- In case of t-inherit-mode="primary", the t-inherit alone does not explain the template structure. For that we need
knowledge of the parent and of the xpaths: the t-name is the only information we need to access that information in the end.

Reply:
@aab-odoo @kebeclibre Lucas proposal: make a third commit that removes the attributes t-inherit t-inherit-mode with a clear message. This will allow an easy revert in case we need those attributes in the web client at some point.

Reply:
Apart from that good job!

Reply:
:+1: